nltk/util.py

```diff
@@ -504,34 +497,16 @@ def ngrams(
     :type sequence: sequence or iter
     :param n: the degree of the ngrams
     :type n: int
-    :param pad_left: whether the ngrams should be left-padded
-    :type pad_left: bool
-    :param pad_right: whether the ngrams should be right-padded
-    :type pad_right: bool
-    :param left_pad_symbol: the symbol to use for left padding (default is None)
-    :type left_pad_symbol: any
-    :param right_pad_symbol: the symbol to use for right padding (default is None)
-    :type right_pad_symbol: any
```

Comment:
Although it's not explicit in the function definition, we shouldn't remove the docs, so that the user know these arguments are use-able. I think it's better to put it back.

Reply:
Done!